<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindShifts</class>
 <widget class="QDialog" name="FindShifts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="p_groupBox_ShiftsData">
     <property name="title">
      <string>Shifts data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_Position">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="p_lbl_StartDate">
        <property name="text">
         <string>Start date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndDate"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="p_comboBox_Positions"/>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartDate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="p_lbl_EndDate">
        <property name="text">
         <string>End date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="p_checkBox_OnlyNotAssigned">
        <property name="text">
         <string>Show only not assigned shifts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="p_groupBox_Result">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="p_treeWidget_Shifts">
        <column>
         <property name="text">
          <string>Shift ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Position</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Start date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>End date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Duration</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
